### ISSUE TITLE
Is it safe to mark rxjs_v5.0.x.js as @flow strict?

### ISSUE THREAD

**Issue by bcherny** [flowtype/flow-typed#2051]

**Comment by GAntoine** [comment]
Pardon my lack of knowledge, what do you mean by "@flow strict"?

**Comment by bcherny** [comment]
Thanks for the quick response @gantoine! See https://github.com/facebook/flow/issues/5776#event-1460714659

**Comment by GAntoine** [comment]
Yeah I don't really think I can give you a good answer. @AndrewSouthpaw thoughts?

**Comment by AndrewSouthpaw** [comment]
¯\_(ツ)_/¯

I'd leave it out until it gets properly documented, unless there's a compelling need to make it strict right now.

**Comment by bcherny** [comment]
@andrewsouthpaw What does “properly documented” mean - like comments and stuff, or a comb through to verify typings?

**Comment by AndrewSouthpaw** [comment]
What I mean is I've not seen any documentation about the meaning of `@flow strict` in the Flow docs, so it'll be hard for others to know what's going on.

Is there a use case for adding `@flow strict` to this libdef?
